afs/afs-ws/afs-ws-storage/src/main/java/com/powsybl/afs/ws/storage/NodeEventClient.java

```diff
@@ -54,6 +70,57 @@ public void onError(Throwable t) {
 
     @OnClose
     public void onClose(Session session) {
-        LOGGER.trace("Node event websocket session '{}' closed for file system '{}'", session.getId(), fileSystemName);
+        int counter = 0;
+        boolean toReconnection = true;
+        long dateInitial = Instant.now().getEpochSecond();
+        while (toReconnection) {
+
+            long dateReconnection = Instant.now().getEpochSecond();
+            if (dateReconnection - dateInitial > config.getReconnectionMax()) {
+                LOGGER.warn("Node event websocket session '{}' closed for file system '{}'", session.getId(), fileSystemName);
```

Comment:
I would move this to error level, mentionning reconnection maximum timeout reached.

Reply:
It's done